etn-ccis/blui-react-workflows
# BLUI-4143 add RegistrationContextProvider

Fixes BLUI-4143.


#### Changes proposed in this Pull Request:
- Add RegistrationContextProvider
- Pair down translations and configure i18n instance within the RegistrationContextProvider
- Add example2 project to run new architecture


#### Screenshots / Screen Recording (if applicable)
-


#### To Test:
- `yarn start:example2`
- The top 2 typography elements should change with the language select buttons to their given translations
- The bottom "test" item is a one-off example of what it looks like to add a single translation to the i18nRegistrationInstance


#### Any specific feedback you are looking for?
-
